src/Symfony/Bundle/FrameworkBundle/Resources/config/object_mapper.php

```diff
@@ -19,6 +19,9 @@
 return static function (ContainerConfigurator $container) {
     $container->services()
         ->set('object_mapper.metadata_factory', ReflectionObjectMapperMetadataFactory::class)
+            ->args([
+                param('object_mapper.map_classes'),
```

Comment:
should we `->ignoreOnInvalid()`? really not sure what the rule is inside the framework bundle for new arguments.